python/Ganga/GPIDev/Lib/File/LCGSEFile.py

```diff
@@ -286,18 +286,13 @@ def getWNInjectedScript(self, outputFiles, indent, patternsToZip, postProcessLoc
 
         return script
 
-    def get(self):
+    def copyTo(self, targetPath):
```

Comment:
Then you're promoting exposing `copyTo` in the baseclass and `copyToUnlessPresent` in each derrived class? This can be done but changes the interface yet again.